srcpkgs/python3-feedparser/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'python3-feedparser'
 pkgname=python3-feedparser
-version=6.0.2
+version=6.0.8
 revision=1
 wrksrc="feedparser-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-sgmllib"
+checkdepends="${depends} python3-pip"
```

Comment:
If you can't fulfill tests with packages from our repos, don't drag pip into it. It makes tests useless, since it won't be using the versions of packages in our own repos.

Reply:
Ah, sorry.  I misinterpreted the check failure (which mentioned pip was missing).  I just needed to add the `depends` to `checkdepends` and it should pass that tests.